pkg/sentry/fsimpl/fuse/fusefs.go

```diff
@@ -210,6 +211,21 @@ type inode struct {
 
 	// the owning filesystem. fs is immutable.
 	fs *filesystem
+
+	// size of the file.
+	size uint64
+
+	// attributeVersion is the version of inode's attributes.
+	attributeVersion uint64
+
+	// attributeTime is the remaining vaild time of attributes.
+	attributeTime uint64
+
+	// version of the inode.
+	version uint64
+
+	// imutex serializes changes to a inode.
```

Comment:
Perhaps the style guide on locks applies here as well https://github.com/google/gvisor/blob/master/g3doc/style.md